### PR TITLE
[337] feat(speaking/talk): modify for 2023 design

### PR DESCRIPTION
**Pull request by SivanYeh**
## Types of changes

* **New feature**

## Description

1.  Apply two-col-wrapper
2. Update 2023 content to the page.

## Steps to Test This Pull Request

1. Go to speaking/talk
2. Check the content in English and Chinese
3. Check the style and see if fit the design on Figma

## Expected behavior

See Figma: https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=39%3A1327&t=bgd3UTBIgUXcPXbT-0

## Related Issue

#337

### CODE REVIEW

File: pages/speaking/talk.vue [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
-                }}</locale-link>
-            </template>
-        </i18n>
+        <i18n path="intro" tag="p"> </i18n>
 
         <!-- Tips -->
-        <i18n path="intro.1" tag="p"></i18n>
         <ul class="list-disc">
             <i18n
                 v-for="(tip, i) in $t('tips')"
```
Comment (yungshenglu) [inline pages/speaking/talk.vue:9]: 需要補一下 `:key`

File: pages/speaking/talk.vue [hunk 2]
```diff
@@ -180,4 +70,9 @@ export default {
 }
 </script>
 
-<style scoped></style>
+<style scoped>
+li,
+p {
+    font-size: 18px;
+}
```
Comment (yungshenglu) [inline pages/speaking/talk.vue:77]: 可以改用 Tailwind 的寫法放進去 `class`

File: pages/speaking/talk.vue (hunk 1 above)
Comment (josix) [reply pages/speaking/talk.vue:9]: 有喔這邊有 `:key="`talk_tips_${i}`"` 在下一行

File: pages/speaking/talk.vue (hunk 2 above)
Comment (josix) [reply pages/speaking/talk.vue]: fix it at `ff9403`

**Review by josix — APPROVED** [review]
LGTM! Thank you, @SivanYeh and @mattwang44 for working on this.

File: pages/speaking/talk.vue (hunk 2 above)
Comment (mattwang44) [reply pages/speaking/talk.vue]: thx!

**Pull request merged**